roadmap/2022/MLOpsRoadmap2022.md

```diff
@@ -40,21 +40,47 @@ Almog Baku, Rimoto <almog.baku@gmail.com>
+
+*[1] A Feature is a specific individual measurable property or characteristic that is used as input to a Trained Model.  By definition, the same Feature must be used as training data and inference input data. This definition intentionally includes both embedded features (e.g., vectors of floating point numbers from transformers, etc models) and traditionally engineered features (e.g.,  transformed or aggregated data using SQL, Python, etc). It includes a systematic link to the required Input Data Sources (schema, etc), any required code (Python, SQL, etc), and any required Trained Models (in the case of an embedding).*
+
+*[2] A Trained Model includes the model artifact (e.g., the model weights or colloquially, the Pickle file), the required Model Server, and a systematic link to the model's required Features.*
+
+*[3] A Model Server includes the compute environment (e.g., Docker containers, K8S YAMLs, etc), any required code or binaries.  A Model Server could range from a simple FastAPI based container, to a Seldon or KServe configuration, or a fully custom serving environment.*
+
+*[4] A Model Training Pipeline includes the compute environment (e.g., Docker containers, K8S YAMLs, etc), model training code, and a systematic link to the Features (packaged as training data) required to train the model.  A Model Training Pipeline could range from a simple Python script to a complex DAG (e.g., Airflow, Kubeflow, Metaflow, etc).*
+ 
+> **MLOps is not to be confused with "AIOps".** AIOps often means an application of AI technologies to help improve the effectiveness of IT or Cloud operations (e.g., AI-based monitoring of logs, etc). This term is still evolving, but for the purposes of this document, we do include AIOps as defined here. Adding to the confusion is that some organisations are more comfortable with the designation 'AI' rather than 'Machine Learning' and thus call their internal 'MLOps" as 'AIOps'. 
 
 ### What is MLOps not?
 
 It sometimes helps to consider what anti-patterns exist around a concept in order to better understand it.
 
-For example, MLOps is not "putting Jupyter Notebooks into production environments".
+**1. MLOps is not "putting Jupyter Notebooks into production environments".**
+
+Rapid Application Development (RAD) tools like Jupyter Notebooks can be extremely useful, both in classroom environments and in exploring problem spaces to understand potential approaches to mathematical problems. However, like all RAD tools, they achieve the rapid element of their name by trading off other key non-functional requirements like maintainability, testability and scalability.  
+
+**2. MLOps is not a purely technical or infrastructure solution**

```

Comment:
I don't agree with this statement. IMO:
> MLOps != ML Infra

As DevOps != Dev Infra (i.e. Kafka and K8s are not sitting under the same category). Although some VCs are confusing between these two subjects, they're not the same.

Reply:
hmm... maybe I understood it wrong? if so - the title is a bit confusing

Reply:
Are you referring to line 81 or line 78-79?